frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -162,14 +240,14 @@ const keywordCompare = (filterString, item) => {
   );
 };
 
-const setURLParams = (params) => {
-  const url = new URL(window.location);
+const setURLParams = (params): void => {
+  const url = new URL(window.location.toString());
```

Comment:
Why do we even do this? why not just use `window.location.hash` and `window.location.pathname` directly?

Reply:
changed to window.location.pathname